src/app/shell/personal-cabinet/provider/create-provider-admin/create-provider-admin.component.html

```diff
@@ -130,7 +131,7 @@ <h3 class="wrapper-title">{{ editMode ? 'РЕДАГУВАННЯ' : 'ДОДАТИ
         <div fxLayout="row" fxLayoutAlign="center center" class="footer">
           <button class="btn" mat-raised-button matStepperPrevious>Назад</button>
           <button mat-raised-button class="btn btn-cancel" (click)="onBack()">Скасувати</button>
-          <button class="btn" mat-button type="submit" (click)="onSubmit()">Зберегти</button>
+          <button class="btn" mat-button type="submit" (click)="onSubmit()" [disabled]="!isWorkshopSelect">Зберегти</button>
```

Comment:
!!managedWorkshopIds.length
I think this is enough and we don't need another one flag in the code